recipes/ocrmypdf/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "ocrmypdf" %}
+{% set version = "11.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocrmypdf-{{ version }}.tar.gz
+  sha256: 0f624456a50be0b0bc8c0b59704d159f637616c093a1cabe8bb383706561bcf7
+
+build:
+  number: 0
+  skip: true   # [py2k or win]
+  entry_points:
+    - ocrmypdf = ocrmypdf.__main__:run
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - cffi >=1.9.1
+    - pip
+    - python
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    - ghostscript
+    - img2pdf
+    - leptonica
+    - pngquant  # [not win]
+    - qpdf  # [not win]
+    - tesseract
+    - unpaper  # [not win]
```

Comment:
Are these all needed, at a glance they look like runtime dependencies.

Reply:
Yes, thank you!